includes/demo-poem.php

```diff
@@ -18,7 +18,7 @@ function poetwp_create_demo_poem() {
 		array(
 			'post_type'      => 'poem',
 			'post_status'    => 'any',
-			'title'          => 'Demo Poem',
+			'title'          => 'Demo Poem: Windswept',
```

Comment:
Why not create a variable (for the title, but also maybe the type and status) so that the values used in the WP_Query are the ones used to save the poem. Use the variable in both places.

Remember, keep the data separate from what you do with it.